advanced-topics/ReactiveUI/src/PrismSample/Views/NugetVersionViewCell.xaml.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Text;
+using System.Threading.Tasks;
+using PrismSample.Converters;
+using PrismSample.ViewModels;
+using ReactiveUI;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace PrismSample.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class NugetVersionViewCell : ContentViewBase<NugetVersionViewModel>
+    {
+        public NugetVersionViewCell()
+        {
+            InitializeComponent();
+
+            this.OneWayBind(ViewModel, x => x.VersionInformation.Version, x => x.PackageVersion.Text, vmToViewConverterOverride: new VersionToStringConverter())
```

Comment:
I would use the lambda expression or globally register.

Reply:
This was meant to be extremely verbose, so you can see there are multiple ways to use the conversion.  I use the lambda expression conversion at other places.  Splat isn't the backing DI engine at the moment, so I didn't want to add the complexity of showing the global registration.

I did add the link to the docs in the README so consumers can look up additional information on the topic.

Reply:
you should probably highlight the recommended ways of doing things... if you're trying to show alternatives it would probably be best to include some comments that explain it...

Reply:
Well.  That was kind of the point of linking the documentation in the README.  The documentation shows all three ways of doing this.  I simply wanted to highlight the two "easiest" ways that don't force you don't the path of DI registrations.  In hopes users would look at the README and read the documentation.

Reply:
Added comments about looking for doc link.